### PR TITLE
chore(chain-ops): migrate contract changes to chain-ops and deploy-config

### PR DESCRIPTION
**Pull request by Pangssu**
Contracts have been added or partially changed for the implementation of multiple validator.
Therefore, I changed the code that generates the L1 genesis of chain-ops, hardhat deploy scripts, and deployment configuration variables.

### CODE REVIEW

File: utils/chain-ops/genesis/config.go
Comment (kangsorang) [inline utils/chain-ops/genesis/config.go:226]: Wouldn't a comparison with the same values as the defaults set in deploy-config be meaningless?

File: utils/chain-ops/genesis/layer_one.go
Comment (seolaoh) [inline utils/chain-ops/genesis/layer_one.go:369]: just a question, why some of predeployed dev L1 contracts addresses are added here, and some other ones are added below `l1Deployer` like [line 450](https://github.com/kroma-network/kroma/pull/56/files#diff-218d848c5297513155401cc5da24b75691a0d263df3c84d99a11bf9913374d95R450)?

File: e2e/e2eutils/secrets.go
Comment (chokobole) [inline e2e/e2eutils/secrets.go:20]: By the way, do we still need this one?

File: utils/chain-ops/genesis/layer_one.go
Comment (chokobole) [reply utils/chain-ops/genesis/layer_one.go:369]: This is because dev L1 contract address remains the same in the production environment as well.

Comment (Pangssu) [reply utils/chain-ops/genesis/layer_one.go:369]: My guess is that l1Deployer passes it directly if the predeployed address is initialized, and deployL1Contracts passes it otherwise. 

In my opinion, if the predeployed addresses are at the beginning of the parameter, these are added in the `l1Deployer`. Otherwise, added in `deployL1Contracts`

Comment (Pangssu) [reply utils/chain-ops/genesis/layer_one.go:369]: In my opinion, if the predeployed addresses are at the beginning of the parameter, these are added in the l1Deployer. Otherwise, added in deployL1Contracts

File: utils/chain-ops/genesis/config.go
Comment (Pangssu) [reply utils/chain-ops/genesis/config.go:226]: AFAIK, default value in deploy-config doesn't affect golang json unmarshaller. So if the value is not set, the default value will be 0.

File: e2e/e2eutils/secrets.go
Comment (Pangssu) [reply e2e/e2eutils/secrets.go:20]: The `CliqueSigner` is required by the L1 geth when running e2e test.

File: utils/chain-ops/genesis/layer_one.go
Comment (seolaoh) [reply utils/chain-ops/genesis/layer_one.go:369]: if there is no reasonable reason, I think it would be better to unify the position of the predeployed address additions between `deployL1Contracts` and `l1Deployer`.

File: packages/contracts/deploy/009-Colosseum.ts
Comment (seolaoh) [inline packages/contracts/deploy/009-Colosseum.ts:25]: are there any reason to omit `CHAIN_ID`, `GUARDIAN`?

**Comment by seolaoh** [comment]
- We need to update `layer_one_test.go`
- in line 462-463 of `layer_one.go`, the comments should be like below.
```go
			/* validatorPool= */ deployment.Args[4].(common.Address),
			/* colosseum= */ deployment.Args[5].(common.Address),
```

File: packages/contracts/deploy/009-Colosseum.ts
Comment (seolaoh) [reply packages/contracts/deploy/009-Colosseum.ts:25]: Cool! BTW, I see you used `portalGuardian` for the security council address, is this temporary until the security council is decided? Or is it possible that guardian for `KromaPortal` and security council for `Colosseum` are the same?

**Review by chokobole — APPROVED** [review]
LGTM, please merge after receiving other's approval.

File: packages/contracts/deploy/009-Colosseum.ts
Comment (Pangssu) [reply packages/contracts/deploy/009-Colosseum.ts:25]: Currently, I think of portal guardian and security council as the same group.

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by 0xHansLee — APPROVED** [review]
lgtm

**Pull request merged**
